ui/src/containers/HtmlHints.svelte

```diff
@@ -5,25 +5,18 @@
     userSession$,
     getIgnoreList
   } from "../stores";
-  import { onMount } from "svelte";
-  import Tabs from "../components/Tabs.svelte";
-  import Icon from "../components/Icon.svelte";
+  import Tabs from "../components/miscComponents/Tabs.svelte";
   import { pipe, map, flatten } from "ramda";
-  import HtmlErrorsTable from "../components/HtmlErrorsTable.svelte";
-  import Breadcrumbs from "../components/Breadcrumbs.svelte";
-  import slug from "slug";
-  import Toastr from "../components/Toastr.svelte";
-  import BuildDetailsCard from "../components/BuildDetailsCard.svelte";
-  import { CONSTS, getPerfScore, HTMLERRORS, printTimeDiff } from "../utils/utils.js";
+  import HtmlErrorsTable from "../components/htmlhintComponents/HtmlErrorsTable.svelte";
+  import Breadcrumbs from "../components/miscComponents/Breadcrumbs.svelte";
+  import Toastr from "../components/miscComponents/Toastr.svelte";
+  import BuildDetailsCard from "../components/detailCard/BuildDetailsCard.svelte";
```

Comment:
The folder names should be consistent.
Change
From
    detailCard
To
    detailCardComponents

Reply:
Done - renamed to detailcardcomponents